infrastructure/php-container/Dockerfile

```diff
@@ -8,4 +8,11 @@ RUN a2enmod include
 # Install pdo_pgsql extension
 RUN apt-get update \
     && apt-get install -y libpq-dev \
-    && docker-php-ext-install pdo_pgsql
+    # One of two PHP extensions (gmp or bcmath) must be installed to perform
+    # cryptographic operations associated with RSA-based JWTs (used by
+    # brick/math via web-token/jwt-core)
+    # See: https://www.php.net/manual/en/book.bc.php
+    # See: https://www.php.net/manual/en/book.gmp.php
+    # See: https://web-token.spomky-labs.com/introduction/pre-requisite
+    # See: https://github.com/brick/math#requirements
+    && docker-php-ext-install pdo_pgsql bcmath
```

Comment:
I'm so worried this will be a showstopper.  😟

Reply:
I'm hopeful between the two we'll get it working 😬  (also, we can always use another pure-php library to rebuild the public key from JWKS data -- this just makes it easier)